src/components/WwwFrame/TheHeader.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<header class="tw-transition-all tw-duration-1000 tw-ease-in-out">
+	<header class="tw-transition-all tw-duration-1000 tw-ease-in-out tw-sticky tw-top-0" style="z-index: 999;">
```

Comment:
We actually only want this for the pages with the ATB feature enabled.

Reply:
That's why it was special-cased before.

Reply:
We have a class for z index.